site/docs/getting-started.md

```diff
@@ -72,7 +72,7 @@ tar -zxvf flink-1.12.7-bin-scala_2.12.tgz
 # 下载 hadoop 依赖
 wget https://repo1.maven.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/${HADOOP_VERSION}-10.0/flink-shaded-hadoop-2-uber-${HADOOP_VERSION}-10.0.jar
 # 下载 arctic flink connector
-wget XXX
+wget https://github.com/NetEase/arctic/releases/download/v0.3.0-rc1/arctic-flink-runtime-1.12-0.3.0.jar
```

Comment:
Shall we make flink 1.12 alternative? 1.14 would be fine.

Reply:
I prefer use specific flink version in quickstart and leave supported flink version description in flink chapter